src/registry/file/edit.tsx

```diff
@@ -0,0 +1,199 @@
+    <Tabs>
+      <TabList>
+        <BuilderTabs.Basic
+          hasErrors={hasAnyError(
+            'label',
+            'key',
+            'description',
+            'tooltip',
+            'showInSummary',
+            'showInEmail',
+            'showInPDF',
+            'multiple',
+            'hidden',
+            'clearOnHide',
+            'isSensitiveData'
+          )}
+        />
+        <BuilderTabs.Advanced hasErrors={hasAnyError('conditional')} />
+        <BuilderTabs.Validation hasErrors={hasAnyError('validate')} />
+        <Tab
+          hasErrors={hasAnyError(
+            'file',
+            'useConfigFiletypes',
+            'of',
+            'fileMaxSize',
+            'maxNumberOfFiles'
+          )}
+        >
+          <FormattedMessage
+            description="Component edit form tab title for 'File' tab"
+            defaultMessage="File"
+          />
+        </Tab>
+        <BuilderTabs.Registration hasErrors={hasAnyError('registration')} />
+        <BuilderTabs.Translations hasErrors={hasAnyError('openForms.translations')} />
+      </TabList>
+
+      {/* Basic tab */}
+      <TabPanel>
+        <Label />
+        <Key isManuallySetRef={isKeyManuallySetRef} generatedValue={generatedKey} />
+        <Description />
+        <Tooltip />
+        <PresentationConfig />
+        <Multiple<FileComponentSchema> />
+        <Hidden />
+        <ClearOnHide />
+        <IsSensitiveData />
+      </TabPanel>
+
+      {/* Advanced tab */}
+      <TabPanel>
+        <SimpleConditional />
+      </TabPanel>
+
+      {/* Validation tab */}
+      <TabPanel>
+        <Validate.Required />
+        <Validate.ValidationErrorTranslations />
+      </TabPanel>
+
+      {/* File tab */}
+      <TabPanel>
+        <FileTabFields />
+      </TabPanel>
+
+      {/* Registration tab */}
+      <TabPanel>
+        <RegistrationTabFields />
+      </TabPanel>
+
+      {/* Translations */}
+      <TabPanel>
+        <Translations.ComponentTranslations<FileComponentSchema>
+          propertyLabels={{
+            label: intl.formatMessage(LABELS.label),
+            description: intl.formatMessage(LABELS.description),
+            tooltip: intl.formatMessage(LABELS.tooltip),
+          }}
+        />
+      </TabPanel>
+    </Tabs>
+  );
+};
+
+/**
+ *
+ * @todo options.withCredentials: true seems to be set somewhere -> session cookie
+ *   needs to be sent by the SDK/client!
+ */
+EditForm.defaultValues = {
+  storage: 'url',
+  url: '',
+  // TODO -> add to type definition!
+  // options: '{"withCredentials": true}',
+  // webcam: false,
+
+  // basic tab
+  label: '',
+  key: '',
+  description: '',
+  tooltip: '',
+  showInSummary: true,
+  showInEmail: false,
+  showInPDF: true,
+  multiple: false,
+  hidden: false,
+  clearOnHide: true,
+  isSensitiveData: true,
+  // Advanced tab
+  conditional: {
+    show: undefined,
+    when: '',
+    eq: '',
+  },
+  // Validation tab
+  validate: {
+    required: false,
+  },
```

Comment:
TODO: remove `plugins` from type definition (it is not supported for file component)